test/stampede_SUITE.erl

```diff
@@ -118,6 +133,8 @@ do_connect(N, Transport, Port, Timeout) ->
 	do_connect(N - 1, Transport, Port, Timeout).
 
 do_log(Pid) when is_pid(Pid) ->
-	logger:info("~p~n", [erlang:process_info(Pid)]);
+	?LOG_INFO("~p: ~p~n", [Pid, erlang:process_info(Pid)]),
```

Comment:
I'd rather see logger:info than an include file and macros.

Reply:
Ok, will see to it on monday :)

Reply:
Ok, done. I had to add some metadata to the call, otherwise it wouldn't log anything, not sure why.

Reply:
You don't need the metadata. But you're in a test suite there so Common Test applies its own rules on top, you might just want to use `ct:log` (if you want them in HTML), `ct:print` (if in the shell, but i wouldn't) or `ct:pal` (both).

Reply:
Ah, learned something new :) Done.